setup.cfg

```diff
@@ -2,10 +2,6 @@
 verbosity=3
 with-doctest=1
 
-[bdist_wheel]
-# This flag says that the code is written to work on both Python 2 and Python 3.
-universal=1
```

Comment:
Are you sure that this flag has no relevance for a Python3-wheel?

Reply:
I am not sure about this, I did this change based on the comment itself. It should be investigated, but IIRC I tried this change with Python 3 and it worked just fine. What is a test scenario that we could try to see if this line is still relevant?